### ISSUE TITLE
Add onit command to execute onos command against onos-config node

### ISSUE THREAD

**Issue by kuujo** [onosproject/onos-config#443]
To ease the inspection of onos-config nodes, it would be convenient to add support for executing an `onos` command against any onos-config node running in onit.

**Comment by Andrea-Campanella** [comment]
@kuujo I was wondering if instead we could expose the IP of the running service/pod of `onos-config` and execute against that ip. Seems like a more "normal" way of doing it.

**Comment by kuujo** [comment]
Yes I'm changing the title and description. This should be implemented as an ingress that can support `onos` and `atomix` CLIs.
